src/views/components/results-table-output.html

```
@@ -2,255 +2,190 @@
-				<div class="results-table_row row-num-{{this.clientSideId}} item-id-{{this.clientSideId}}"
+		<table class="results-table {{#if targetsCnt === 1}}single-target{{/if}}">
+			<thead class="results-table_header">
+				<tr>
+					<th class="results-table_header_cell location" scope="col">
+						<button type="button" on-click="@this.onSortClick('location')">
+							Location
+							<img width="8" alt="" src="{{@shared.assetsHost}}/img/{{@this.getSortIcon('location')}}">
+						</button>
+					</th>
+
+					{{#if targetsCnt > 1}}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)